CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+## 1.0.0
```

Comment:
should probably just be unreleased, when the release is created this ver num will be created